.env

```diff
@@ -73,6 +73,7 @@ _APP_EXECUTOR_SECRET=your-secret-key
 _APP_EXECUTOR_HOST=http://proxy/v1
 _APP_FUNCTIONS_RUNTIMES=php-8.0,node-18.0,python-3.9,ruby-3.1
 _APP_MAINTENANCE_INTERVAL=86400
+_APP_MAINTENANCE_DELAY=20
```

Comment:
Lets document this variable in the variables.php file as well as add it in the compose.phtml file